src/GLPINetwork.php

```diff
@@ -145,7 +145,7 @@ public static function getGlpiUserAgent(): string
         $comments = sprintf('installation-mode:%s', GLPI_INSTALL_MODE);
         if (!empty(GLPI_USER_AGENT_EXTRA_COMMENTS)) {
            // append extra comments (remove '(' and ')' chars to not break UA string)
-            $comments .= '; ' . preg_replace('/\(\)/', ' ', GLPI_USER_AGENT_EXTRA_COMMENTS);
+            $comments .= '; ' . preg_replace('/\(\)/', ' ', GLPI_USER_AGENT_EXTRA_COMMENTS)[0];
```

Comment:
`GLPI_USER_AGENT_EXTRA_COMMENTS` is expected to be a string, not an array, so the result of `preg_replace` is expected to be a string too.

```suggestion
            $comments .= '; ' . (string)preg_replace('/\(\)/', ' ', GLPI_USER_AGENT_EXTRA_COMMENTS);
```